TTS/tts/utils/synthesis.py

```diff
@@ -219,6 +219,7 @@ def synthesis(model,
             ap (TTS.tts.utils.audio.AudioProcessor): audio processor to process
                 model outputs.
             speaker_id (int): id of speaker
+            style_wav (str | Dict[str, float]): Uses for style embedding of GST.
             style_wav (str): Uses for style embedding of GST.
```

Comment:
this line is a duplicate of the above,

Reply:
> PS2 : As you can see, I changed some packages requirements. On conda, the old requierements.txt was having no error on python 3.6 and 3.7. But on pip, both 36 and 37 were having errors. With the change I've made, py3.7 is now not having errors. However, py3.6 + pip is still having errors on these new requirements.

I don't see the requirements changed in this PR ?

Reply:
Ok. As it can also be a dict, should I remove the old one to only let the new one ? (str or dict of str|float ?)